### ISSUE TITLE
Expanders are presented as "group collapsed" in Narrator instead of presenting Name/HelpText

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/terminal#12317]
### Windows Terminal version

1.13.10321.0

### Windows build number

22548.1002

### Other Software

None

### Steps to reproduce

1. Open Narrator or Accessibility Insights
2. Navigate to an Expander in the Settings UI (i.e. "Startup" > "When Terminal starts")

### Expected Behavior

The setting name/description should be exposed as the AutoProp.Name/HelpText respectively.

### Actual Behavior

Name/HelpText are undefined.

**Comment by carlos-zamora** [comment]
This will be fixed after updating to MUX 2.8 because it was fixed in https://github.com/microsoft/microsoft-ui-xaml/pull/6032.

**Comment by zadjii-msft** [comment]
Got a build with 2.8. This wasn't fixed.

**Comment by zadjii-msft** [comment]
Duping this up to #13019, since that's the a11y tracking thread.

/dup #13019
